rest-api-spec/src/main/resources/rest-api-spec/api/indices.open.json

```diff
@@ -34,6 +34,10 @@
            "options" : ["open","closed","none","all"],
            "default" : "closed",
            "description" : "Whether to expand wildcard expression to concrete indices that are open, closed or both."
+        },
+        "wait_for_active_shards": {
+          "type" : "string",
+          "description" : "Sets the number of shard copies that must be active before proceeding with the index operation. Defaults to 1, meaning the primary shard only. Set to `all` for all shard copies, otherwise set to any non-negative value less than or equal to the total number of copies for the shard (number of replicas + 1)"
```

Comment:
I think you copied this from the wrong location (this is the description that applies to the indexing command, not the index creation command). That one just states

```
"wait_for_active_shards": {
          "type" : "string",
          "description" : "Set the number of active shards to wait for before the operation returns." 
        },
```